lib/conekta-button.scss

```diff
@@ -1,242 +1,207 @@
+          }
+        }
+      }
+    }
+  }
 }
 
-.#{$modal_name}-body .card{
-	background: #F9FAFB;
-	border: solid 2px #E2E6E9;
-	padding: 15px;
-	border-radius: 4px;
-	margin-top: 14px;
-	margin-bottom: 35px;
-	.info{
-		position: relative;
-		padding-bottom: 20px;
-
-		.tag{
-			font-size: 14px;
-			color: #FFF;
-			border-radius: 10px;
-			background-color: #838991;
-			display:inline-block;
-			padding-top: 3px;
-			padding-bottom: 3px;
-			padding-left: 6px;
-			padding-right: 6px;
-
-		}
-
-		.brand-cards {
-			float: right;
-			display:inline-block;
-			margin-bottom: 15px;
-			margin-top: 5px;
-
-			> div{
-				display: inline-block;
-				padding-right: 5px;
-				padding-left: 5px;
-			}
-		}
-
-	}
-
-	
-
-
-	.group{
-		display: inline-block;
-		position: relative;
-		min-height: 1px;
-
-		&.number{
-			width: 188px;
-			margin-right: 4.5px;
-		}
-
-		&.date{
-			width: 127px;
-			margin-left: 4.5px;
-		}
-
-		&.name{
-			width: 240px;
-			margin-right: 4.5px;
-
-		}
-
-		&.cvc{
-			width: 75px;
-			margin-left: 4.5px;
-		}
-
-		label{
-			font-size: 12px;
-			color: #838991;
-		}
-
-		input{
-			margin-top: 10px;
-			margin-bottom: 15px;
-			height:38px;
-			padding-top: 8px;
-			padding-bottom: 8px;
-			padding-left: 12px;
-			font-size: 14px;
-			font-family: $font-family;
-			width: 100%;
-			color: darken(#fff, 85%);
-
-			border: solid 1px  #DADFE2;
-			border-radius: 4px;
-
-			&::focus{
-				border: solid 1px  #DADFE2;
-				outline: 0;
-
-			}
-
-			&::-webkit-input-placeholder {
-			    color:    #939C9F;
-			}
-
-			&::-moz-placeholder {
-			    color:    #939C9F;
-			}
-
-			&::-moz-placeholder {
-			    color:    #939C9F;
-			}
-
-			&:-ms-input-placeholder {
-			    color:    #939C9F;
-			}
-		}
-	}
-
-
+.#{$modal_name}-body .card {
+  background    : #F9FAFB;
+  border        : 2px solid #E2E6E9;
+  border-radius : 4px;
+  margin        : 14px 0 35px;
+  padding       : 15px;
+
+  .info {
+    padding-bottom : 20px;
+    position       : relative;
+
+    .tag {
+      background-color : #838991;
+      border-radius    : 10px;
+      color            : #FFF;
+      display          : inline-block;
+      font-size        : 14px;
+      padding          : 3px 6px;
+    }
+
+    .brand-cards {
+      display : inline-block;
+      float   : right;
+      margin  : 5px 0 15px;
+
+      > div {
+        display : inline-block;
+        padding : 0 5px;
+      }
+    }
+  }
+
+  .group {
+    display    : inline-block;
+    min-height : 1px;
+    position   : relative;
+
+    &.number {
+      margin-right : 4.5px;
+      width        : 188px;
+    }
+
+    &.date {
+      margin-left : 4.5px;
+      width       : 127px;
+    }
+
+    &.name {
+      margin-right : 4.5px;
+      width        : 240px;
+
+    }
+
+    &.cvc {
+      margin-left : 4.5px;
+      width       : 75px;
+    }
+
+    label {
+      color     : #838991;
+      font-size : 12px;
+    }
+
+    input {
+      border        : 1px solid #DADFE2;
```

Comment:
Color `#DADFE2` should be written as `#dadfe2`<br>Property name should be immediately followed by a colon